src/app/(BottomTabNavigation)/Cases/CaseScreen/EligibilityForm/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { View, Text, Image } from 'react-native';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import { useLocalSearchParams, useRouter } from 'expo-router';
+import styles from './styles';
+import { updateCaseStatus } from '../../../../../supabase/queries/cases';
+import { CaseUid, Eligibility } from '../../../../../types/types';
+import Arrow from '../../../../../../assets/arrow.svg';
+
+export default function EligibilityForm() {
+  const router = useRouter();
+  const { caseId } = useLocalSearchParams<{
+    caseId: CaseUid;
+  }>();
+  return (
+    <View style={styles.container}>
+      <View style={styles.textContainer}>
+        <View style={styles.insideContainer}>
+          <View>
+            <TouchableOpacity
+              onPress={() => router.back()}
+              style={styles.backButton}
+            >
+              <Text>Go Back</Text>
+            </TouchableOpacity>
+          </View>
+          <View>
+            <Text>The eligibility requirements are as follows:</Text>
+          </View>
+          <View>
+            <Text>Sexy</Text>
+            <Text>Tall</Text>
+            <Text>Handsome</Text>
+          </View>
+          <View>
+            <Text>Do you meet the following requirements?</Text>
+          </View>
+        </View>
+      </View>
+      <View style={styles.buttonsContainer}>
+        <View style={styles.buttonWrapperTop}>
+          <TouchableOpacity
+            style={[styles.button, styles.buttonTop]}
+            onPress={() => updateCaseStatus(caseId, Eligibility.ELIGIBLE)}
```

Comment:
`updateCaseStatus` is an async function so let's use a combination of `useEffect` and `useState` to update eligibility. We can have an `eligibility` state variable and a `useEffect` that calls `updateCaseStatus` and sets this variable!

Reply:
I'm looking into routing with this right now and running into annoying issues (just wanted to let you know but don't worry about that right now). The only way I can confirm that this is correct right now is by going back to the `Cases` screen with the Case Cards and seeing that the "File Claim" and "Opt Out" buttons appear if the status is correct. This with a mix of logging.